api/api.go

```diff
@@ -416,6 +424,11 @@ type CloudBackupInfo struct {
 	Metadata map[string]string
 	// Status indicates the status of the backup
 	Status string
+	// Cluster indicates if the cloudbackup was uploaded by this
+	// cluster. Could be unknown with older version cloudbackups
+	Cluster SdkCloudBackupClusterType
+	// Namespace to which this cloudbackup belongs to
```

Comment:
is the namespace of the source volume ? 
Also, I would name it ClusterType instead of Cluster for clarity.

Reply:
yes. k8s namespace that source volume belongs to.